From 24de6373c51138f082ae8b5d27a301f53567fb9a Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Fri, 8 Nov 2024 23:33:40 -0500
Subject: [PATCH 3/4] Uprev level-zero-npu-extensions to build against headers
 > 1.18

Note that this patch is not packaged, instead the submodule code is directly
patched with the extra commit on Fedora >= 41 only, since 40 is still
shipping OneAPI Level Zero headers 1.17.n.

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 third_party/level-zero-npu-extensions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/level-zero-npu-extensions b/third_party/level-zero-npu-extensions
index a63155a..110f48e 160000
--- a/third_party/level-zero-npu-extensions
+++ b/third_party/level-zero-npu-extensions
@@ -1 +1 @@
-Subproject commit a63155ae4e64feaaa6931f4696c2e2e699063875
+Subproject commit 110f48ee8eda22d8b40daeeecdbbed0fc3b08f8b
-- 
2.45.2

